zallet/src/config.rs

```diff
@@ -142,6 +146,31 @@ impl BuilderSection {
     }
 }
 
+/// Indexer configuration section.
+#[derive(Clone, Debug, Default, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct IndexerSection {
+    /// Full node / validator listen port.
+    pub validator_listen_address: Option<SocketAddr>,
```

Comment:
This field was identified as being badly named in the Zaino Community signal chat; we should do a better job here (it's not an address that Zaino listens on.)
```suggestion
    pub validator_address: Option<SocketAddr>,
```

Reply:
It's not; it's the address that the validator (Zebra) listens on. Unless you mean that there's potential for confusion about this being an address that Zallet listens on?